specs/multibool.test.ts

```diff
@@ -0,0 +1,32 @@
+import { describe, it } from './_describe.ts'
+import { or } from '../mod.ts'
+import { eq } from './utils/utils.ts'
+import { and } from '../mod.ts'
+import { xor } from '../mod.ts'
+
+describe('mutlibool', () => {
+  it(' Should solve (a + b) . (c + d. e + (f + g)) ', () => {
+    eq(
+      and(
+        or([], ''),
+        or(or(' ', and(false, { a: 31 })), or(true, NaN)),
```

Comment:
Still less readability.
Store temporary results in variables.
Also, comment the boolean function demonstrated.
You can use pipe and compose functions. 

For example
```typescript
// Expression - (2*5+5-10)/2
const double = multiply(2)
const half = divide(_, 2)
const add5 = add(5)
const subtract10 = subtract(_, 10)

half(subtract10(add5(double(5))))           // 2.5
compose(half, subtract10, add5, double)(5)  // 2.5
pipe(double, add5, subtract10, half)(5)     // 2.5
```